src/views/Accessibility/AccessibiltyRequest/Create/index.tsx

```diff
@@ -1,34 +1,212 @@
 /* eslint-disable react/prop-types */
-import React from 'react';
-import assert from 'assert';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+import { Button, ComboBox } from '@trussworks/react-uswds';
+import {
+  Field as FormikField,
+  Form as FormikForm,
+  Formik,
+  FormikProps
+} from 'formik';
 import CreateAccessibilityRequestQuery from 'queries/CreateAccessibilityRequestQuery';
-import * as yup from 'yup';
+import GetSystemsQuery from 'queries/GetSystems';
+import { GetSystems } from 'queries/types/GetSystems';
 
-import { MutationField, MutationForm } from 'components/MutationForm';
-import { CreateAccessibilityRequestInput } from 'types/graphql-global-types';
-
-const CreateAccessibilityRequestFormSchema = yup.object({
-  name: yup
-    .string()
-    .trim()
-    .required('Enter a name for this request')
-    .default('')
-});
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageWrapper from 'components/PageWrapper';
+import PlainInfo from 'components/PlainInfo';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import HelpText from 'components/shared/HelpText';
+import Label from 'components/shared/Label';
+import { NavLink, SecondaryNav } from 'components/shared/SecondaryNav';
+import TextField from 'components/shared/TextField';
+import { initialAccessibilityRequestFormData } from 'data/accessibility';
+import { AccessibilityRequestForm } from 'types/accessibility';
+import flattenErrors from 'utils/flattenErrors';
+import accessibilitySchema from 'validations/accessibilitySchema';
 
 const Create = () => {
-  // Move this into the MutationForm?
-  const req: CreateAccessibilityRequestInput = CreateAccessibilityRequestFormSchema.getDefaultFromShape();
-  assert(req);
+  const history = useHistory();
+  const { t } = useTranslation('accessibility');
+  const { data } = useQuery<GetSystems>(GetSystemsQuery, {
+    variables: {
+      // TODO: Is there a way to make this all? or change the query?
+      first: 20
+    }
+  });
+
+  const [mutate, mutationResult] = useMutation(CreateAccessibilityRequestQuery);
+  const handleSubmit = (values: AccessibilityRequestForm) => {
+    mutate({
+      variables: {
+        input: {
+          name: values.requestName,
+          intakeID: values.intakeId
+        }
+      }
+    }).then(() => {
+      history.push('/', {
+        confirmationText: `${values.requestName} was added to the 508 requests page`
+      });
+    });
+  };
+
+  const systems = useMemo(() => {
```

Comment:
Using `useMemo` here and blow so the component isn't mapping over the data every re-render. This is probably a _really_ minor performance gain.

I did it primarily to avoid any bugs with the `ComboBox`. In order to avoid any strange behavior, I want to make sure we're passing the same array into the ComboBox options rather than pass in new ones every re-render.